src/riscv-hybrid-integration.adoc

```diff
@@ -88,6 +88,14 @@ CRE=1 for the current privilege level
 
 When the <<m_bit>> can be set follows the rules defined by <<ACPERM>>.
 
+[#m_bit_observe,reftext="Observing the M-bit"]
+==== Observing the M-bit
```

Comment:
Could you please rename the section to something like `querying the CHERI execution mode`? Please note that the methods here don't actually tell you what the M-bit of PCC is, but rather what is the _effective_ CHERI mode which depends on PCC and a handful of CSRs.